tensorflow/lite/micro/examples/magic_wand/sparkfun_edge/accelerometer_handler.cc

```diff
@@ -132,13 +139,13 @@ TfLiteStatus SetupAccelerometer(tflite::ErrorReporter* error_reporter) {
   }
 
   if (lis2dh12_fifo_mode_set(&dev_ctx, LIS2DH12_BYPASS_MODE)) {
-    TF_LITE_REPORT_ERROR(error_reporter, "Failed to clear FIFO buffer.");
-    return 0;
+    error_reporter->Report("Failed to clear FIFO buffer.");
+    return kTfLiteError;
   }
 
   if (lis2dh12_fifo_mode_set(&dev_ctx, LIS2DH12_DYNAMIC_STREAM_MODE)) {
-    TF_LITE_REPORT_ERROR(error_reporter, "Failed to set streaming mode.");
-    return 0;
+    error_reporter->Report("Failed to set streaming mode.");
```

Comment:
same as above `TF_LITE_REPORT_ERROR should be used instead of error_reporter, so log strings can be removed to save space if needed.`